datadog-agent-variables.tf

```diff
@@ -0,0 +1,51 @@
+variable "datadog_agent_enabled" {
+  type    = bool
+  default = true
+}
+
+variable "datadog_agent_evaluation_period" {
+  type    = string
+  default = "last_5m"
+}
+
+variable "datadog_agent_note" {
+  type    = string
+  default = ""
+}
+
+variable "datadog_agent_docs" {
+  type    = string
+  default = ""
+}
+
+variable "datadog_agent_filter_override" {
+  type    = string
+  default = ""
+}
+
+variable "datadog_agent_alerting_enabled" {
+  type    = bool
+  default = true
+}
+
+variable "datadog_agent_no_data_timeframe" {
+  type    = number
+  default = null
+}
+
+variable "datadog_agent_notify_no_data" {
+  type    = bool
+  default = false
+}
+
+variable "datadog_agent_ok_threshold" {
+  type    = number
+  default = null
+}
+
+variable "datadog_agent_priority" {
+  description = "Number from 1 (high) to 5 (low)."
+
+  type    = number
+  default = 2
+}
```

Comment:
This is pet peeve of mine: prefer adding a newline at the end of a file. It prevents messy diffs later on. Having a newline at the of the line as a default make sense since tools like `wc -l` report wrong statistics otherwise.

There is no need to fix this now though.